### DISCUSSION TITLE
Song loop countdown

### DISCUSSION THREAD

**Discussion by s0q** [sircharlo/meeting-media-manager#91] [Ideas] [answered]
We typically play music for 5 minutes before the program starts, could you add a timer slider or something next to the song loop button so that when you press it, it plays for 5 minutes and starts fading out after the set amount of time has elapsed?

**Comment by sircharlo** [comment]
Good idea. I'll think something up before next month's planned release.

**Reply by sircharlo** [reply]
Ideally, in addition to the possibility of a "dumb" play-for-x-minutes timer, I'd also like to implement a feature that, when enabled, would:
- intelligently get the next probable meeting start time (determine the next :00 or :30), 
- trigger the music fade out a predetermined x number of seconds before that time 

Those who remember Soundbox will know what I'm talking about. 😄

**Reply by sircharlo** [reply]
Initial implementation in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/0d6b96368f6cb32f54bae9cf0740e7fda19641c6

**Comment by sircharlo** [comment, marked as answer]
See #100
